base/set.jl

```diff
@@ -103,6 +116,30 @@ const ⊆ = issubset
 ⊊(l::Set, r::Set) = <(l, r)
 ⊈(l::Set, r::Set) = !⊆(l, r)
 
+doc"""
+    unique(C,k::Integer)
+
+Returns up to k distinct values in C in the order they appear
```

Comment:
and signature should be added into the rst docs so genstdlib populates the docstring correctly